bitnami/fluentd/values-production.yaml

```diff
@@ -274,10 +274,10 @@ aggregator:
     ## Service ports
     ##
     ports:
-      # http:
-      #   port: 9880
-      #   targetPort: http
-      #   protocol: TCP
+      http:
```

Comment:
Why did you put it as default value?

Reply:
This port was already specified in the [containerPorts](https://github.com/irizzant/charts-1/blob/271dfd8b8cd014ede5beb293487ffcf20d8e5797/bitnami/fluentd/values-production.yaml#L95), I added it to the Service ports to reflect this so that Istio doesn't detect it as an unknown endpoint and it doesn't trigger Push warnings.